### PR TITLE
feat(charts): set default fstype to ext4 in csi-provisioner

### PR DESCRIPTION
**Pull request by shovanmaity**
Set default fstype to ext4 in csi-provisioner. This will be helpful when fsType is not mention in storageclass.

### CODE REVIEW

File: deploy/lvm-operator.yaml [hunk 1]
```diff
@@ -1182,6 +1182,7 @@ spec:
             - "--strict-topology"
             - "--leader-election"
             - "--extra-create-metadata=true"
+            - "--default-fstype=ext4"
```
Comment (pawanpraka1) [inline deploy/lvm-operator.yaml:1185]: This file is autogenerated, you need to make the same change in deploy/yamls/zfs-driver.yaml

**Review by pawanpraka1 — APPROVED** [review]
looks good.

File: deploy/lvm-operator.yaml (hunk 1 above)
Comment (shovanmaity) [reply deploy/lvm-operator.yaml:1185]: done

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Pull request merged**